aclk/aclk_lws_wss_client.c

```diff
@@ -5,378 +5,375 @@
-	struct lws_wss_packet_buffer *tail = *list;
-	if(!*list) {
-		*list = item;
-		return;
-	}
-	while(tail->next) {
-		tail = tail->next;
-	}
-	tail->next = item;
+    struct lws_wss_packet_buffer *tail = *list;
+    if (!*list) {
+        *list = item;
+        return;
+    }
+    while (tail->next) {
+        tail = tail->next;
+    }
+    tail->next = item;
 }
 
 static inline struct lws_wss_packet_buffer *lws_wss_packet_buffer_pop(struct lws_wss_packet_buffer **list)
 {
-	struct lws_wss_packet_buffer *ret = *list;
-	if(ret != NULL)
-		*list = ret->next;
+    struct lws_wss_packet_buffer *ret = *list;
+    if (ret != NULL)
+        *list = ret->next;
 
-	return ret;
+    return ret;
 }
 
 static inline void lws_wss_packet_buffer_free(struct lws_wss_packet_buffer *item)
 {
-	freez(item->data);
-	freez(item);
+    freez(item->data);
+    freez(item);
 }
 
 static inline void _aclk_lws_wss_read_buffer_clear(struct lws_ring *ringbuffer)
 {
-	size_t elems = lws_ring_get_count_waiting_elements(ringbuffer, NULL);
-	if(elems > 0)
-		lws_ring_consume(ringbuffer, NULL, NULL, elems);
+    size_t elems = lws_ring_get_count_waiting_elements(ringbuffer, NULL);
+    if (elems > 0)
+        lws_ring_consume(ringbuffer, NULL, NULL, elems);
 }
 
 static inline void _aclk_lws_wss_write_buffer_clear(struct lws_wss_packet_buffer **list)
 {
-	struct lws_wss_packet_buffer *i;
-	while((i = lws_wss_packet_buffer_pop(list)) != NULL) {
-		lws_wss_packet_buffer_free(i);
-	}
-	*list = NULL;
+    struct lws_wss_packet_buffer *i;
+    while ((i = lws_wss_packet_buffer_pop(list)) != NULL) {
+        lws_wss_packet_buffer_free(i);
+    }
+    *list = NULL;
 }
 
-static inline void aclk_lws_wss_clear_io_buffers(struct aclk_lws_wss_engine_instance *inst)
+static inline void aclk_lws_wss_clear_io_buffers()
 {
-	aclk_lws_mutex_lock(&inst->read_buf_mutex);
-	_aclk_lws_wss_read_buffer_clear(inst->read_ringbuffer);
-	aclk_lws_mutex_unlock(&inst->read_buf_mutex);
-	aclk_lws_mutex_lock(&inst->write_buf_mutex);
-	_aclk_lws_wss_write_buffer_clear(&inst->write_buffer_head);
-	aclk_lws_mutex_unlock(&inst->write_buf_mutex);
+    aclk_lws_mutex_lock(&engine_instance->read_buf_mutex);
+    _aclk_lws_wss_read_buffer_clear(engine_instance->read_ringbuffer);
+    aclk_lws_mutex_unlock(&engine_instance->read_buf_mutex);
+    aclk_lws_mutex_lock(&engine_instance->write_buf_mutex);
+    _aclk_lws_wss_write_buffer_clear(&engine_instance->write_buffer_head);
+    aclk_lws_mutex_unlock(&engine_instance->write_buf_mutex);
 }
 
-static const struct lws_protocols protocols[] = {
-	{
-		"aclk-wss",
-		aclk_lws_wss_callback,
-		sizeof(struct aclk_lws_wss_perconnect_data),
-		0, 0, 0, 0
-	},
-	{ NULL, NULL, 0, 0, 0, 0, 0 }
-};
+static const struct lws_protocols protocols[] = { { "aclk-wss", aclk_lws_wss_callback,
+                                                    sizeof(struct aclk_lws_wss_perconnect_data), 0, 0, 0, 0 },
+                                                  { NULL, NULL, 0, 0, 0, 0, 0 } };
```

Comment:
seems in this case it is less readable?

Reply:
I can't remember the change but I can guess what happened. The `clang-format` doesn't have a setting for vertical alignment of elements inside structures. It bin-packs them instead so if you had this arranged in a more readable way then the linter removed it. I prefer the vertical alignment personally, but the team "agrees" on the `clang-format` settings...

Reply:
I wouldn't say that the team "agrees" :).